### PR TITLE
New package: i3wsr-1.2.2

### PR DESCRIPTION
**Pull request by steinex**

### CODE REVIEW

File: srcpkgs/i3wsr/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'i3wsr'
+pkgname=i3wsr
+version=1.2.2
+revision=1
+build_style=cargo
+hostmakedepends="python3-devel"
```
Comment (ndowens) [inline srcpkgs/i3wsr/template]: -devel files should go in makedepends, any particular reason it is in hostmakedepends?

Comment (steinex) [reply srcpkgs/i3wsr/template]: build needs python3 which won't work for cross when just in makedepends.

Comment (ndowens) [reply srcpkgs/i3wsr/template]: Just tried it without and yes you are correct, but also just set it to python3 and it went fine, so just try with python3 in hostmake instead.

Comment (steinex) [reply srcpkgs/i3wsr/template]: done

**Pull request merged**